### PR TITLE
Avoid creating release archives in RPM builds

### PR DESCRIPTION
**Pull request by smarterclayton**
It saves time copying binaries, and we don't actually use the release binary downstream anymore.

### CODE REVIEW

File: hack/build-cross.sh [hunk 1]
```diff
@@ -82,21 +82,30 @@ os::build::build_binaries "${OS_CROSS_COMPILE_TARGETS[@]}"
...
-OS_RELEASE_ARCHIVE="openshift-origin" \
-  os::build::place_bins "${OS_CROSS_COMPILE_BINARIES[@]}"
-
-# Make the image binaries release.
-OS_BUILD_PLATFORMS=("${image_platforms[@]+"${image_platforms[@]}"}")
-OS_RELEASE_ARCHIVE="openshift-origin-image" \
-  os::build::place_bins "${OS_IMAGE_COMPILE_BINARIES[@]}"
-
-os::build::release_sha
+if [[ "${OS_BUILD_RELEASE_ARCHIVES-}" != "n" ]]; then
```
Comment (csrwng) [inline hack/build-cross.sh]: "n" may be confusing, esp if we don't use it anywhere else in the hack scripts. Why not 1/0 or even true/false and make the default false (as in OS_SKIP_BUILD_RELEASE_ARCHIVES)

Comment (smarterclayton) [reply hack/build-cross.sh]: It's consistent with other flags we have like this.

Comment (smarterclayton) [reply hack/build-cross.sh]: Maybe we're not consistent across our existing flags though.

Comment (smarterclayton) [reply hack/build-cross.sh]: I'll take a todo to do a pass on flags when i split out the generic hack/* stuff for Ben.

Comment (stevekuznetsov) [reply hack/build-cross.sh]: Been meaning to do this for a while:

```sh
function os::util::as_boolean() {
	local var="$1"

	if [[ "${var}" == 'false' || "${var}" == 'f' || "${var}" == 'no' || "${var}" == 'n' || "${var}" == '0' || -z "${var}" ]]; then
		return 1
	elif [[ "${var}" == 'true' || "${var}" == 't' || "${var}" == 'yes' || "${var}" == 'y' || "${var}" == '1' ]]; then
		return 0
	else
		os::log::fatal "Value \`${var}\` could not be interpreted as a boolean."
	fi
}
```

Comment (stevekuznetsov) [reply hack/build-cross.sh]: Biggest issue is all the places where set == true and unset == false right now. Any non-null false value would not be backwards compat there

**Pull request merged**